matsim/src/main/java/org/matsim/core/config/groups/SubtourModeChoiceConfigGroup.java

```diff
@@ -36,6 +36,8 @@ public final class SubtourModeChoiceConfigGroup extends ReflectiveConfigGroup {
 	public final static String MODES = "modes";
 	public final static String CHAINBASEDMODES = "chainBasedModes";
 	public final static String CARAVAIL = "considerCarAvailability";
+	public final static String SINGLE_PROBA = "probaForRandomSingleTripMode";
+
```

Comment:
further down, the default value is set to 0. In a comment within the same line, it is said that this is done in order to maintain backwards compatibility but should normally be changed. I think this is the point in time where we should consider to conduct the change. Might break a few tests, though.
On the other hand, you incorporated at least a warning the the VspConfigConsistencyChecker